networkx/classes/multigraph.py

```diff
@@ -253,6 +256,31 @@ def __init__(self, data=None, **attr):
         self.edge_key_dict_factory = self.edge_key_dict_factory
         Graph.__init__(self, data, **attr)
 
+    def new_edge_key(self, u, v):
+        """Return an unused key for edges between nodes `u` and `v`.
+
+        The nodes `u` and `v` do not need to be already in the graph.
+
+        In the standard graph class the new key is the number of existing
+        edges between `u` and `v` (increased if necessary to ensure unused).
+
+        Parameters
+        ----------
+        u, v : nodes
+
+        Returns
+        -------
+        key
```

Comment:
key is an integer here

Let's add a notes section that explains that the keys start at 0 and are incremented by 1.  Perhaps we should also note that if edges are removed from the graph that the key new key will no longer be the number of edges between the two nodes for any removed edges.